webtau-core-groovy/src/main/groovy/org/testingisdocumenting/webtau/persona/PersonaExtensions.groovy

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 webtau maintainers
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.testingisdocumenting.webtau.persona
+
+import java.util.function.Supplier
+
+class PersonaExtensions {
+    static <R> R call(Persona persona, Closure code) {
+        return persona.execute(code as Supplier<R>)
```

Comment:
Why a different name?  In the http extensions at least we've preserved the names.

Reply:
not following

Reply:
The `call` vs `execute` thing.

Reply:
`call` is a groovy magic method -- this is what let's you do 
```
John {
}
```

Reply:
Ah, ok